### PR TITLE
failing test for #207

### PR DESCRIPTION
**Pull request by efx**

### CODE REVIEW

File: tests/parse-result-test.js [hunk 1]
```diff
@@ -1211,7 +1221,7 @@ QUnit.module('ember-template-recast', function() {
 
     QUnit.test('mutations retain custom whitespace formatting', function(assert) {
       let template = stripIndent`
-        <Foo
```
Comment (rwjblue) [inline tests/parse-result-test.js]: Please revert this change, this whitespace is intentional

File: tests/parse-result-test.js [hunk 2]
```diff
@@ -22,6 +22,16 @@ QUnit.module('ember-template-recast', function() {
       assert.equal(print(ast), `<img src="{{this.something}}">`);
     });
 
+    QUnit.test('test descriptions with carets pass', function(assert) {
+      let template = `it('<' function() {
```
Comment (rwjblue) [inline tests/parse-result-test.js]: I don't really understand what this is testing. This project does not support parsing javascript files...

Comment (efx) [reply tests/parse-result-test.js]: @rwjblue I created a test here as it was the simplest way to demonstrate the error I saw while running the angle brackets codemod on integration tests in my app. It looks like someone has posted an issue too: https://github.com/ember-codemods/ember-angle-brackets-codemod/issues/240

Comment (efx) [reply tests/parse-result-test.js]: https://github.com/ember-template-lint/ember-template-recast/issues/207 has my initial breakdown if that helps
